extensions/dicom-p10-downloader/src/index.js

```diff
@@ -0,0 +1,43 @@
+import { getDicomWebClientFromConfig } from './utils';
+import { getCommands } from './commandsModule';
+
+/**
+ * Constants
+ */
+
+/**
+ * Globals
+ */
+
+const sharedContext = {
+  dicomWebClient: null,
+};
+
+/**
+ * Extension
+ */
+export default {
+  /**
+   * Only required property. Should be a unique value across all extensions.
+   */
+  id: 'dicom-p10-downloader',
+
+  /**
+   * LIFECYCLE HOOKS
+   */
+
+  preRegistration({ appConfig }) {
+    const dicomWebClient = getDicomWebClientFromConfig(appConfig);
```

Comment:
So if the server ever changes, this will stop working? (not that we can currently change servers, just curious)

Reply:
Actually, this is just a "default" DICOM Web Client instance built based on startup config which is only used if the used does not provide its own DICOMWebClient instance. Also, the command from extensions/dicom-p10-downloader/src/commandsModule.js file (line #11, listed above) uses the information inside the servers store to find out information about the currently active server and builds a DICOM Web Client from there.